templates/server/src/main/resources/archetype-resources/core/src/main/java/__packageInPathFormat__/general/logic/impl/UcManageBinaryObjectImpl.java

```diff
@@ -48,20 +48,20 @@ public BinaryObjectEto saveBinaryObject(Blob data, BinaryObjectEto binaryObjectE
   @Override
   public void deleteBinaryObject(Long binaryObjectId) {
 
-    this.binaryObjectRepository.delete(binaryObjectId);
+    this.binaryObjectRepository.deleteById(binaryObjectId);
 
   }
 
   @Override
   public BinaryObjectEto findBinaryObject(Long binaryObjectId) {
 
-    return getBeanMapper().map(this.binaryObjectRepository.findOne(binaryObjectId), BinaryObjectEto.class);
+    return getBeanMapper().map(this.binaryObjectRepository.find(binaryObjectId), BinaryObjectEto.class);
```

Comment:
As it seems the spring-data guys have renamed `findOne` to `getOne`. After things have been spread out to the wild these breaking API changes will not amuse the projects I assume. We should consider this for https://github.com/devonfw/devcon/issues/135